scheduler/src/test/scala/com/sky/kms/e2e/SchedulerIntSpec.scala

```diff
@@ -48,7 +49,10 @@ class SchedulerIntSpec extends SchedulerIntSpecBase {
           cr.key should contain theSameElementsInOrderAs schedule.key
           cr.value should contain theSameElementsInOrderAs schedule.value.get
```

Comment:
As `random[ScheduledEvent]` can generate the `value` field as `None` around 10% of the time, it causes the right hand side of this test to fail. Investigating a possible fix for this.